sql/opt_trace.cc

```diff
@@ -712,12 +712,26 @@ void print_best_access_for_table(THD *thd, POSITION *pos)
   DBUG_ASSERT(thd->trace_started());
 
   Json_writer_object obj(thd, "chosen_access_method");
-  obj.
-    add("type", pos->type == JT_ALL ? "scan" : join_type_str[pos->type]).
-    add("rows_read", pos->records_read).
-    add("rows_out", pos->records_out).
-    add("cost", pos->read_time).
-    add("uses_join_buffering", pos->use_join_buffer);
+
+  obj.add("type", pos->type == JT_ALL ? "scan" : join_type_str[pos->type]);
+
+  if (pos->type == JT_EQ_REF || pos->type == JT_REF ||
+      pos->type == JT_MAYBE_REF || pos->type == JT_FT)
```

Comment:
I grep for `JT_MAYBE_REF` and this value doesn't seem to be ever used. Please remove it from here.

Reply:
Wait, also fix the commit comment: